### ISSUE TITLE
CopyQ won't open

### ISSUE THREAD

**Issue by tbird301** [hluk/CopyQ#1377]
System requirements shown at MacUpdate where 10.6.8 or higher. I use OS10.9.5

Installed CopyQ, launched it & got this message:
CopyQ cannot be opened because of a Problem.
Check with the developer to make sure CopyQ works with this version of OSX. You may need to reinstall the application.  Be sure to install any available updates for the application and OS X.
Sounds to me that the posted system requirements posted at MacUpdate are outdated.  Or is it still 10.6.8?

**Comment by hluk** [comment]
I don't use macOS nor am I familiar with MacUpdate. Is this issue for CopyQ v3.11.0 installed using brew or directly from the official builds?

The official builds are available at: https://github.com/hluk/CopyQ/releases

The build logs for v3.11.0 are available at: https://travis-ci.org/github/hluk/CopyQ/jobs/681831953

**Comment by tbird301** [comment]
Hi, 
 
Did not use Homebrew.

**Comment by hluk** [comment]
I don't know what to fix here. If that helps, the app is built and bundled on Travis CI. The build log says:

```
+ /Users/travis/build/hluk/CopyQ/build/_CPack_Packages/Darwin/DragNDrop/copyq--Darwin/CopyQ.app/Contents/MacOS/CopyQ --version
CopyQ Clipboard Manager v3.11.0
Qt: 5.13.2
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: macOS High Sierra (10.13)
```

**Comment by tbird301** [comment]
Thank you so much.  Now I know that the OS needs to be 10.13.

**Comment by hluk** [comment]
I don't think it needs to be exactly 10.13 but I'm not sure what the exact OS version requirements are. Any ideas how to find this info?

**Comment by tbird301** [comment]
What this means is that OS 10.13 is the minimum system requirements.  Usually when the system requirements are posted it will usually look like this:

App requirements: 
Intel 64
macOS 10.12.0 or later

**Comment by szigya** [comment]
There was an issue before where someone wrote that it was after qt (?) update, so doesn't matter if you use homebrew or build from source. With this you can fallback to previous versions:

```
 #Remove if already installed
brew remove copy
brew cleanup
#Install qt
brew install qt5
tldr brew
brew install cmake extra-cmake-modules
git clone https://github.com/hluk/CopyQ.git
cd CopyQ
git checkout v3.13.0
cmake -DCMAKE_PREFIX_PATH="$(brew --prefix qt5)" .
cmake --build .
cpack
ls
cp CopyQ.app /Applications
```

**Comment by Chealer** [comment]
@tbird301: Which version do you use, and what do you mean by “MacUpdate”?

**Comment by hluk** [comment]
This is very old, let me close that. If there is still a problem please provide new info.

**Comment by XhstormR** [comment]
fix the issues:
```
xattr -d com.apple.quarantine /Applications/CopyQ.app
codesign --force --deep --sign - /Applications/CopyQ.app
```
